internal/framework/service/turnstile/schema.go

```diff
@@ -65,6 +65,9 @@ func (r *TurnstileWidgetResource) Schema(ctx context.Context, req resource.Schem
 				Validators: []validator.String{
 					stringvalidator.OneOf("world"),
 				},
+				PlanModifiers: []planmodifier.String{
```

Comment:
I'm not clear on why we're keeping this `"region"` schema attribute, but that's a tf question not a blocker.

change otherwise LGTM

Reply:
I left it in because the create has it as an option.

https://developers.cloudflare.com/api-next/resources/turnstile/subresources/widgets/methods/create/

Reply:
Ahhhh I see what you mean now! LGTM++